nlprule/src/bin/run.rs

```diff
@@ -1,28 +1,29 @@
-use clap::Clap;
-use nlprule::{rules::Rules, tokenizer::Tokenizer};
+// use clap::Clap;
+// use nlprule::{rules::Rules, tokenizer::Tokenizer};
 
-#[derive(Clap)]
-#[clap(
-    version = "1.0",
-    author = "Benjamin Minixhofer <bminixhofer@gmail.com>"
-)]
-struct Opts {
-    text: String,
-    #[clap(long, short)]
-    tokenizer: String,
-    #[clap(long, short)]
-    rules: String,
-}
+// #[derive(Clap)]
+// #[clap(
+//     version = "1.0",
+//     author = "Benjamin Minixhofer <bminixhofer@gmail.com>"
+// )]
+// struct Opts {
+//     text: String,
+//     #[clap(long, short)]
+//     tokenizer: String,
+//     #[clap(long, short)]
+//     rules: String,
+// }
 
-fn main() {
-    env_logger::init();
-    let opts = Opts::parse();
+fn main() {}
+// fn main() {
+//     env_logger::init();
+//     let opts = Opts::parse();
 
-    let tokenizer = Tokenizer::new(opts.tokenizer).unwrap();
-    let rules = Rules::new(opts.rules).unwrap();
+//     let tokenizer = Tokenizer::new(opts.tokenizer).unwrap();
+//     let rules = Rules::new(opts.rules).unwrap();
 
-    let tokens = tokenizer.pipe(&opts.text);
+//     let tokens = tokenizer.pipe(&opts.text);
 
-    println!("Tokens: {:#?}", tokens.collect::<Vec<_>>());
-    println!("Suggestions: {:#?}", rules.suggest(&opts.text, &tokenizer));
-}
+//     println!("Tokens: {:#?}", tokens.collect::<Vec<_>>());
+//     println!("Suggestions: {:#?}", rules.suggest(&opts.text, &tokenizer));
+// }
```

Comment:
Are you planning on re-using this? If not, it tends to be cleaner in the long run to delete + commit and revert as needed than to comment + commit.